Gemfile

```diff
@@ -50,6 +50,10 @@ gem 'bootsnap', require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️

Reply:
Thanks, I will take this tip for the next project.